report_thesis/src/sections/results/optimization_results.tex

```diff
@@ -0,0 +1,82 @@
+\subsection{Optimization Results}\label{sec:optimization_results}
+In this section, we present and analyze the results of running the Optimization experiment that we described in Section~\ref{subsec:optimization_experiment_design}.
+As mentioned, our primary objective was to identify the optimal configurations for predicting the concentration of various oxides in our dataset.
+We systematically evaluated a range of machine learning models, preprocessing techniques, and hyperparameter settings to determine the most effective combinations for each oxide.
+
+The results of the experiment were ~$16.000$ trials worth of data on the configurations used, hyperparameters, as well as metrics.
+
+Our data cleaning for this dataset primarily included filtering out failed runs, which was caused by configurations that did not work well together, as well as filtering out extreme error values.
+We filter out any runs that had a \gls{rmsecv} above 50.
+Approaches like \gls{svr} could occasionally yield this kind of outlier result in specific configurations.
+We selected 50 as threshold to include as many trials that were not obvious outliers.
+
+Our experiment underwent mostly without issues.
+Some issues are to be expected given the scale of the experiment.
+Unfortunately, we encountered an issue with a server we were using, resulting in some oxides and models having to be re-run.
```

Comment:
What kind of issue? Should we specify?

Reply:
Nah too much technical stuff, would require a looong explanation of the process for running, how it interacts with other servers, etc.